### DISCUSSION TITLE
Interface Language at version 3.0.5

### DISCUSSION THREAD

**Discussion by fernandozucatelli** [texstudio-org/texstudio#1477] [Q&A] [answered]
### Language issue

After updating TeXstudio to version 3.0.5 I notice a small bug with the language settings.

I change the Language at "Options ->Configure TeXstudio -> General -> Appearance -> Language" from default to any other language, but the system only change to English. Sometimes it changes to German, since German is the standard language of the machine I'm using.

It doesn't seem to be an issue related to the dictionaries, because the language checker is working correctly in each language I tested.

**Comment by sunderme** [comment, marked as answer]
this is fixed. Please install 3.1.0.

**Reply by fernandozucatelli** [reply]
Ok. I downloaded it and it worked perfectly. Thanks for the fast update.
